Makefile

```diff
@@ -76,6 +76,9 @@ build:: build_proto build_display_wasm .make/ensure/go
 build_display_wasm:: .make/ensure/go
 	cd pkg && GOOS=js GOARCH=wasm go build -o ../bin/pulumi-display.wasm -ldflags "-X github.com/pulumi/pulumi/sdk/v3/go/common/version.Version=${VERSION}" ./backend/display/wasm
 
+build_local:: build_proto .make/ensure/go
+	export GOBIN=$(shell realpath ./bin) && make dist
```

Comment:
I thought all the tests pulled from PATH? Since when did ./bin get used for anything.

Reply:
They do, but the idea here is that we don't necessarily have to pollute what's on the PATH every time we run the integration tests. https://github.com/pulumi/pulumi/pull/18453/files/10cc896ecd2b3d4f958c1bfb95b6eb3b9894e5d9#diff-0a2a73f63f0707d0366654971055ac58350fa92558376fe46e93406052d86464R51 puts $repo/bin first in the PATH, so it will pull `pulumi` and the language plugins from there, instead of the global `pulumi` installation from the user.

Reply:
ah nice I follow